src/cookbooks/active_dau.md

```diff
@@ -24,6 +24,21 @@ ORDER BY
     1 ASC
 ```
 
+[`clients_last_seen`](../datasets/bigquery/clients_last_seen/reference.md) can also be used, but is only available in BigQuery. Below is an example query for getting aDAU and aMAU using `clients_last_seen`.
+
+```sql
+SELECT
+    submission_date,
+    COUNTIF(days_since_visited_5_uri < 1) AS visited_5_uri_dau,
+    COUNTIF(days_since_visited_5_uri < 28) AS visited_5_uri_mau
+FROM
+    telemetry.clients_last_seen
+GROUP BY
+    1
+ORDER BY
+    1 ASC
```

Comment:
I'd prefer to be verbose and name columns in the group by and order by clauses (so `submission_date` rather than 1) for docs and for production ETL.

Reply:
I agree, but I wanted to match the existing style in the document